client/my-sites/plans/current-plan/current-plan-thank-you/backup-thank-you.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThankYou from './thank-you';
+
+const BackupProductThankYou = ( { translate } ) => (
+	<ThankYou
+		illustration="/calypso/images/illustrations/security.svg"
+		showCalypsoIntro
+		showContinueButton
+		title={ translate( 'Hello backups!' ) }
+	>
+		<Fragment>
+			<p>{ translate( 'We just finished settup up backups for you.' ) }</p>
+			<p>
+				{ translate( 'Next, we’ll take a look at your new WordPress.com dashboard. ' ) }
+				{ translate( 'You can manage your backups under “Activity” in the sidebar. ' ) }
```

Comment:
Since "Activity" is nested under the "Tools" menu (which is collapsed by default in this view), should we clarify where the "Activity" item is? Something like "Tools > Activity"?